src/App.css

```diff
@@ -0,0 +1,42 @@
+.App {
+  text-align: center;
+}
+
+body {
+  background-image: url("./img/korktavle.png");
+  background-position: cover;
+  overflow-x: hidden; 
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
```

Comment:
Det er bedre å pakke en font med websida i stedet for å liste opp 100 stykk og håpe at brukeren har en av dem...